addOns/commonlib/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added AbstractHostFilePlugin for use with ElmahScanRule and other future Host level file scan rules (Issue 6133).
 - Maintenance changes (Issue 6376 & Issue 6099).
+- Added DiceMatcher and related test, which implements the Dice algorithm to calculate the percentage similarity between two strings.
```

Comment:
The part of the test is not that relevant (they are not included in the add-on).